curs/chap-04-reprezentare/01-data-interp/data_interp.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+unsigned char c[8] = {0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77, 0x88};
+unsigned short s[4];
+unsigned int i[2];
+unsigned long long ll[1];
```

Comment:
I would use more descriptive names for these variables, such as `char_array`, `short_array` etc. Since it's a demo we can sacrifice efficiency of typing for comprehensiveness.